Utills/Utillty.dart

```diff
@@ -0,0 +1,75 @@
+import 'dart:io';
+import '../Functions/AddRecord.dart';
+import '../Functions/deleteUserData.dart';
+import '../Functions/saveUserDetailsLocally.dart';
+import '../Functions/displayUserDetails.dart' as userdata;
+import '../Functions/fetchData.dart';
+class UtilityIO {
+
+  // getting data from databse
+  static Map<int,Map<String,dynamic>> getDatabaseData(){
+    Map<int,Map<String,dynamic>> data = readingFile();
+
+    return data;
+
+  }
+  // saving data to database
+  static void setDatabaseData(Map<int,Map<String,dynamic>> data){
+      saveUSerDetails(data);
+  }
+
+  static Map<int,Map<String,dynamic>> deleteData(Map<int,Map<String,dynamic>> data){
+   data = deleteRecord(data);
+   return data; 
+  }
+
+  static Map<int,Map<String,dynamic>> addNewData(Map<int,Map<String,dynamic>> data){
+    data = addingRecording(data);
+    return data; 
+  }
+
+  static void displayData(Map<int,Map<String,dynamic>> data){
+    userdata.displayData(data);
+  }
+
+
+  // Use for Input
+  static String InputUtility(Map<int,Map<String,dynamic>> data){
```

Comment:
function names always in camelCase, also note that it should be verb. 
`readInput`.
Also return type can be void as not returning